openapi.yaml

```diff
@@ -9702,21 +9750,28 @@ components:
             helm_chart: 0.0.15
           name: production-2020-03-26-W9J3uzHl
           code:
-            prs: 1
-            lines_prs: 37
-            lines_overall: 1200
-            commits_prs: 2
-            commits_overall: 20
+            prs:
+              'repo-0': 32
```

Comment:
Can you please chacnge to `github.com/athenianco/athenian-webapp` and `github.com/athenianco/athenian-api`

Reply:
done